apps/dapp-console/app/settings/components/LinkedWallet.tsx

```diff
@@ -0,0 +1,68 @@
+import { shortenAddress } from '@eth-optimism/op-app'
+import {
+  Button,
+  Dialog,
+  DialogContent,
+  DialogTrigger,
+  Input,
+} from '@eth-optimism/ui-components'
+import { Text } from '@eth-optimism/ui-components/src/components/ui/text'
+import { WalletWithMetadata } from '@privy-io/react-auth'
+import { RiAlertFill, RiCloseLine } from '@remixicon/react'
+import { useCallback, useState } from 'react'
+
+export type LinkedWalletProps = {
+  wallet: WalletWithMetadata
+  onUnlink: (wallet: WalletWithMetadata) => void
+}
+
+export const LinkedWallet = ({ wallet, onUnlink }: LinkedWalletProps) => {
+  const [isDialogOpen, setIsDialogOpen] = useState(false)
+
+  const handleUnlink = useCallback(() => onUnlink(wallet), [wallet, onUnlink])
+  const handleCancel = useCallback(
+    () => setIsDialogOpen(false),
+    [setIsDialogOpen],
+  )
+
+  return (
+    <div className="flex flex-row gap-2">
+      <Input className="rounded" value={wallet.address} disabled />
+      <Dialog open={isDialogOpen} onOpenChange={setIsDialogOpen}>
+        <DialogTrigger asChild>
+          <Button className="rounded" variant="secondary" size="icon">
```

Comment:
q: do we need `aria-label` when button text is not clear (maybe it is clear inside <RiCloseLine>)

Reply:
yea i think semantically you do want to use an `aria-label` when a button just contains svg, will add!